assets/js/pages/Auth.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from "react"
+import { useNavigate } from "react-router-dom"
+
+const Auth = () => {
+
+    const navigate = useNavigate();
+    const redirectToHomePage = () => navigate("/");
+
+    const fetchToken = (google_response) => {
+        const request = {   // MAG: it seems like a common type - can I import it from somewhere?
+            method: 'POST',
+            headers: { 'Content-Type': 'application/json' },
+            body: JSON.stringify(google_response),
+        };
+
+        fetch("http://localhost:4000/auth/login", request)  //MAG: how should I store this endpoint uri? 
```

Comment:
Maybe it is time to think about env variables. Then it might be set as http://localhost:4000 for development and xyx for production ;)

Reply:
It's changed to `window.location.origin`. I think it's enough for now